webapp/channels/src/components/advanced_text_editor/formatting_bar/formatting_bar.tsx

```diff
@@ -278,6 +278,9 @@ const FormattingBar = (props: FormattingBarProps): JSX.Element => {
                                 className='control'
                                 onClick={makeFormattingHandler(mode)}
                                 disabled={disableControls}
+                                {...!showHiddenControls && {
```

Comment:
Although this solution works, what could be a more usefull solution would be to not render the popover/options itself it the first place `webapp/channels/src/components/advanced_text_editor/formatting_bar/formatting_bar.tsx` already uses floating ui, take a look at the `webapp/channels/src/components/profile_popover/profile_popover_controller.tsx` where we are using the `show` state to hide/show the popover itself, can you make this changes please